task/04-arrays-tasks.js

```diff
@@ -131,41 +147,46 @@ function getUpperCaseStrings(arr) {
  *    [ 'angular', 'react', 'ember' ] => [ 7, 5, 5 ]
  */
 function getStringsLength(arr) {
-   throw new Error('Not implemented');
-}
+   return arr.reduce((newArr, item) => {
+         newArr.push(item.length);
+         return newArr;
+      }, []); ;
+   }  
 
 /**
- * Inserts the item into specified array at specified index
+ * Inserts the item into specified    array at specified index
  * 
  * @param {array} arr
  * @param {any} item
  * @param {number} index 
  * 
  * @example
  *    [ 1, 3, 4, 5 ], 2, 1  => [ 1, 2, 3, 4, 5 ]
- *    [ 1, 'b', 'c'], 0, 'x'  => [ 'x', 1, 'b', 'c' ]
+ *    [ 1, 'b', 'c'], 'x', 0,   => [ 'x', 1, 'b', 'c' ]
  */
 function insertItem(arr, item, index) {
-   throw new Error('Not implemented');
+   let secondPartOfArr = arr.splice(index, 1000000, item);
+   arr.push(...secondPartOfArr);
+   return arr;
```

Comment:
Some dirty and wrong logic.
```
let secondPartOfArr = arr.splice(index, 0, item);
  return secondPartOfArr;
```

second argument (0) at splice method shows, that we don't delete any element